cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.cpp

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfBotPawn.h"
+
+#include <Camera/CameraComponent.h>
+#include <Components/InputComponent.h>
+
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Unreal.h"
+#include "UrdfParser.h"
```

Comment:
Although this is not strictly required, we should use a fully-qualified `#include "UrdfBot/UrdfParser.h"` for consistency with the rest of the code. Likewise with all of the other local includes in this module.

Reply:
fixed